todo_handler/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+class TaskCategory(models.Model):
+    name = models.CharField(max_length=100)
+    
+    def __str__(self):
+        return self.name
+    
+class Task(models.Model):
+    STATUS_CHOICES = (
```

Comment:
Use text choice
https://stackoverflow.com/questions/54802616/how-can-one-use-enums-as-a-choice-field-in-a-django-model